overrides/components/EIC-related-contents.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "$veda-ui/react";
+import styled from "$veda-ui/styled-components";
+import { media, glsp } from "$veda-ui/@devseed-ui/theme-provider";
+import { stories } from "veda";
+import Hug from "$veda-ui-scripts/styles/hug";
+import {
+  Card,
+  CardList
+} from "$veda-ui-scripts/components/common/card";
+import { ContentsPropType } from './theme-card';
+
+const CardWrapper = styled(Hug)`
+  padding: ${glsp(1, 0)};
+  grid-column: 1/-1;
+`;
+
+const StyledCard = styled(Card)`
+  figure {
+    height: 200px;
+  }
+`
+
+const EvenCardList = styled(CardList)`
+  ${media.largeUp`
+    grid-template-columns: repeat(2, 1fr);
+  `}
+`
+
+export default function EICRelatedContents({ storyIds }:ContentsPropType) {
```

Comment:
I was trying to prefix the component with `eic` because we already have a component called `related-content` on veda-ui level, but you are right that it ends up being ambiguous name. I changed it to `related-content-cards`